modulefiles/clas12root/1.8.5

```diff
@@ -0,0 +1,10 @@
+#%Module1.0
+
+prereq hipo/4.1.0
+prereq_optional qadb/3.0.0
+prereq_optional iguana/0.8.0
```

Comment:
both of these are optional dependencies, but these are the minimum versions needed for this clas12root release; is this the correct syntax?

Reply:
i think it only supports lists, so, when there's a new,compatible one, we add it, e.g., https://github.com/JeffersonLab/clas12-env/blob/048863f8778e6814e9bafdb7f04918ba954847c7/modulefiles/iguana/0.5.0#L3